src/armscan_env/envs/labelmaps_navigation.py

```diff
@@ -168,8 +168,8 @@ def get_optimal_action_array(self) -> np.ndarray:
         full_action_arr = self.get_full_optimal_action_array()
         return full_action_arr[self._get_projected_action_arr_idx()]
 
-    def step_to_optimal_state(self) -> None:
-        self.step(self.get_optimal_action())
+    def step_to_optimal_state(self) -> tuple[TObs, float, bool, bool, dict[str, Any]]:
```

Comment:
TObs -> Observation[LabelmapStateAction, Any]

Reply:
I have the same return as in the step function, is this not correct?

Reply:
It is not, because step is defined inside the generic base class, whereas this method is defined in the non-generic implementation